packages/core/react-loosely-lazy/src/lazy/__tests__/lazy-client.unit.test.tsx

```diff
@@ -313,6 +314,8 @@ describe('lazy* on the client', () => {
           </LazySuspense>
         );
 
+        await nextTick();
```

Comment:
This is now needed as we use `requestIdleCallback` so preloads are added on next tick